### PR TITLE
cyf-hotel-Halit

### PR DESCRIPTION
**Pull request by halitos**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+      <TourisrticInfoCardsContainer />
       <Bookings />
+      <Footer
+        adress={[
+          "123 Fake Street, London, E1 4UD",
+          "hello@fakehotel.com",
+          "0123 456789"
+        ]}
+      />
     </div>
```
Comment (Ashaghel) [inline src/App.js:21]: Well done

File: src/Footer.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+const Footer = props => {
+  return (
+    <div className="nav flex-column align-items-center bg-secondary text-white p-3 mt-5">
+      <h4>Contact Details</h4>
+      <ul className="nav flex-column">
+        {props.adress.map((item, index) => {
+          return (
+            <li className="nav-item" key={index}>
+              {item}
+            </li>
```
Comment (Ashaghel) [inline src/Footer.js:12]: good work!

File: src/SearchResults.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          <th scope="col">E-mail</th>
+          <th scope="col">Room No</th>
+          <th scope="col">Check-in Date</th>
+          <th scope="col">Check-out Date</th>
+          <th scope="col">Numer of Nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map((p, index) => {
+          return (
```
Comment (Ashaghel) [inline src/SearchResults.js:22]: great job ,

File: src/SearchResults.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            <tr key={index}>
+              <td>{p.id}</td>
+              <td>{p.title}</td>
+              <td>{p.firstName}</td>
+              <td>{p.surname}</td>
+              <td>{p.email}</td>
+              <td>{p.roomId}</td>
```
Comment (Ashaghel) [inline src/SearchResults.js:29]: those cells don't have an Id or anything to use later on, do you want to add something here ?

File: src/SearchResults.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+          <th scope="col">Room No</th>
+          <th scope="col">Check-in Date</th>
+          <th scope="col">Check-out Date</th>
+          <th scope="col">Numer of Nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map((p, index) => {
+          return (
+            <tr key={index}>
```
Comment (Ashaghel) [inline src/SearchResults.js:23]: putting key here may not be the best way 
read more 
https://reactjs.org/docs/lists-and-keys.html

Comment (halitos) [reply src/SearchResults.js]: I believe the problem is using index here as a key. So I assigned id numbers as keys.

**Pull request closed without merging**